crates/forge/assets/vyper/VyperDeployerTemplate.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {Vm} from "forge-std/Vm.sol";
+
+contract VyperDeployer {
```

Comment:
I believe it is possible, per https://github.com/foundry-rs/foundry/pull/8181, to `deployCode` directly like this https://github.com/mpeyfuss/vyper-template/blob/08a0eb515b2f0493d595de3ebf5a048442ea9a0f/script/Counter.s.sol#L10

eliminating the need for `VyperDeployer` and `FFI` should also not be required anymore